src/components/regions-map/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect, useContext, useRef } from 'react'
+
+import { ResponsiveChoropleth } from '@nivo/geo'
+import { getStateByDescription, getStateDescription } from 'helpers/dictionary'
+import { ThemeContext } from 'styled-components'
+import BrasilGeoJSON from '../../helpers/brasil_estados.json'
+
+import * as S from './styles'
+import Tooltip from 'components/tooltip'
+
+interface Props {
+  data: Array<{ name: string; total: number }>
```

Comment:
You use this `{ name: string; total: number }` twice only in this file, could you create an interface for this?